book.rb

```diff
@@ -0,0 +1,12 @@
+require 'rental'
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+    @person = nil
+  end
+
+  attr_accessor :title, :author, :rentals, :person
```

Comment:
- [ ] **REQUIRED:** It will be nice to create a method for adding rental through a many-to-many relationship
[Link to example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)